src/models/DownloadLimitAction.js

```diff
@@ -48,8 +48,8 @@ export default class DownloadLimitAction {
 			icon: 'icon-download',
 			is: this._store.enabled ? ActionInput : null,
 			text: t('files_downloadlimit', 'Download limit'),
-			title: t('files_downloadlimit', 'Download count: {count}', this._store),
-			value: this._store.limit,
+			title: t('files_downloadlimit', 'Downloads: {count}', this._store),
+			value: this._store.limit ? this._store.limit - this._store.count : this._store.limit,
```

Comment:
I am not sure this change makes sense, the label says download limit so it should show the limit, not something else.

Reply:
It makes sense because the remaining downloads is the current download limit. To show the initial download limit one week later to the customer does not deliver any new information to the user because he may remember the initial download limit or can calculate it with the download counter. It is more helpful to tell the user the current download limit to enable him to decide if he wants to increase it or not.